src/AppLovinMAXAdView.js

```diff
@@ -1,7 +1,25 @@
-import { requireNativeComponent, UIManager, findNodeHandle } from "react-native";
+import { NativeModules, requireNativeComponent, UIManager, findNodeHandle } from "react-native";
 import PropTypes from "prop-types";
 import React from "react";
-import AppLovinMAX from "./index.js";
+
+const { AppLovinMAX } = NativeModules;
+
+export const AdFormat = {
```

Comment:
Are we breaking: https://github.com/AppLovin/AppLovin-MAX-React-Native/blob/5ba78824bd68334e5fd35f64d99658351cad3071/example/src/App.tsx#L324 ?

Reply:
No, it is exported again through 'export default' in index.js.

```
export default {
  ...AppLovinMAX,
  AdView,
  userSegment,
  targetingData,
  AdContentRating,
  UserGender,
  ConsentDialogState,
  AdViewPosition, 
  AdFormat,                     <== here
  addEventListener,
  removeEventListener,
  ...
```